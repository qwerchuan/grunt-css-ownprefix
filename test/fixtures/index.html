<div class="foot btn">

</div>
