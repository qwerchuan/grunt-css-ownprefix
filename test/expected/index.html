<div class="self-foot self-btn">

</div>
